concrete-core/src/backends/core/private/math/fft/transform.rs

```diff
@@ -43,7 +43,8 @@ impl Fft {
     pub fn new(size: PolynomialSize) -> Fft {
         debug_assert!(
             [128, 256, 512, 1024, 2048, 4096, 8192, 16384].contains(&size.0),
```

Comment:
I can see that this list is used in many locations in this PR. Do you think it could make sense to define a `const ALLOWED_POLY_SIZE: [usize;8]` in `fft/mod.rs` and then refer to it in the rest of the codebase ?

Reply:
Sounds good.